sentry-sidekiq/lib/sentry/sidekiq/sentry_context_middleware.rb

```diff
@@ -95,7 +95,10 @@ def call(worker_class, job, queue, _redis_pool)
 
         user = Sentry.get_current_scope.user
         job["sentry_user"] = user unless user.empty?
-        job["trace_propagation_headers"] ||= Sentry.get_trace_propagation_headers
+
+        if Sentry.configuration.sidekiq.propagate_traces
+          job["trace_propagation_headers"] ||= Sentry.get_trace_propagation_headers
+        end
```

Comment:
Isn't this a breaking change in a way? Previously it would do it unconditionally. Now you have to explicitly enable it. I understand that the original behavior was problematic though so maybe it's fine?

Reply:
it defaults to true